src/Public/js/nipsweb.managelibrary.js

```diff
@@ -122,34 +138,35 @@ var Library = function () {
                     var track_album = "";
                     var track_position = "";
 
-                    // Build a list of tracks from the lastfm responses and store it in a drop
-                    // down list
-                    var select = "";
-                    if (!manual_track) {
-                        select = $('<select></select>')
-                        .attr('name', response.fileid).attr('id', 'centralupload-' + i);
-                        $.each(
-                            response.analysis,
-                            function (
-                                key,
-                                value
-                            ) {
-                                select.append('<option value="' + value.title + ':-:' + value.artist + '">' + value.title + ' by ' + value.artist + '</option>');
+
+                    if (manual_track) {
```

Comment:
Seems to be quite a lot of code that only does stuff if manual_track is set. Can some early returns be used to reduce the indentation?